src/test/java/io/kestra/plugin/microsoft365/sharepoint/ExportIT.java

```diff
@@ -0,0 +1,171 @@
+package io.kestra.plugin.microsoft365.sharepoint;
+
+import com.microsoft.graph.serviceclient.GraphServiceClient;
+import io.kestra.core.junit.annotations.KestraTest;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIf;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStream;
+import java.net.URI;
+import java.util.ArrayList;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@KestraTest
+@EnabledIf(
+    value = "io.kestra.plugin.microsoft365.sharepoint.ExportIT#shouldRunIntegrationTests",
+    disabledReason = "Missing required environment variables: AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, SHAREPOINT_SITE_ID, SHAREPOINT_DRIVE_ID"
+)
+class ExportIT {
+    private static final Logger log = LoggerFactory.getLogger(ExportIT.class);
+
+    @Inject
+    private RunContextFactory runContextFactory;
+
+    private RunContext runContext;
+    private java.util.List<String> createdItemIds;
```

Comment:
Avoid to have shared items like createdItemIds, driveId and parentId in your tests... Please limit the test class attributes to the minimum 🙏 Here and everywhere.